torch/fx/passes/shape_prop.py

```diff
@@ -45,20 +43,27 @@ def _extract_tensor_metadata(result : torch.Tensor) -> TensorMetadata:
             break
 
     is_quantized = result.is_quantized
-    qscheme = None
+    qparams = None
     q_scale = None
     q_zero_point = None
+    q_per_channel_scales = None
+    q_per_channel_zero_points = None
+    q_per_channel_axis = None
 
     if is_quantized:
-        qscheme = result.qscheme()
-
-        if qscheme in {torch.per_tensor_affine, torch.per_tensor_symmetric}:
-            q_scale = result.q_scale()
-            q_zero_point = result.q_zero_point()
-
+        qparams = {}
+        qparams["qscheme"] = result.qscheme()
+        qparams["dtype"] = result.dtype  # type: ignore[assignment]
```

Comment:
What error are these `# type: ignore` comments suppressing? Can we type qparams as `Dict[str, Any]` and make this go away?